.buildkite/steps/test-demo-notebooks.sh

```diff
@@ -27,13 +27,11 @@ f=${NOTEBOOKS[$INDEX]}
 case $(basename "$f") in
   'attacks_clustering_analysis.ipynb' | 'hateful-twitters-interpretability.ipynb' | 'hateful-twitters.ipynb' | 'stellargraph-attri2vec-DBLP.ipynb' | \
     'node-link-importance-demo-gat.ipynb' | 'node-link-importance-demo-gcn.ipynb' | 'node-link-importance-demo-gcn-sparse.ipynb' | 'rgcn-aifb-node-classification-example.ipynb' | \
-    'calibration-pubmed-link-prediction.ipynb' | 'ensemble-link-prediction-example.ipynb' | 'movielens-recommender.ipynb' | 'stellargraph-metapath2vec.ipynb' | 'stellargraph-node2vec-weighted-random-walks.ipynb' | 'calibration-pubmed-node-classification.ipynb' | \
-    'ensemble-node-classification-example.ipynb' | 'cora-gcn-links-example.ipynb')
+    'calibration-pubmed-link-prediction.ipynb' | 'ensemble-link-prediction-example.ipynb' | 'movielens-recommender.ipynb' | 'stellargraph-metapath2vec.ipynb' | 'stellargraph-node2vec-weighted-random-walks.ipynb' | 'calibration-pubmed-node-classification.ipynb' | 'ensemble-node-classification-example.ipynb')
```

Comment:
Did you mean to also remove `ensemble-node-classification-example.ipynb` here since you've added the dataset downloading? Or it still won't be running in CI?

Reply:
I first ran to make sure the notebook works, but it's now skipped here since it's too slow - listed in #820